charts/portal/values-production.yaml

```diff
@@ -795,6 +795,12 @@ ingress-nginx:
 #  tcp:
 #    9443: "<namespace>/dispatcher:9443"
 
+#Settings for portal jobs
+jobs:
```

Comment:
jobs didn't have its own values section so labels were done under "portal" section.
e.g. https://github.com/CAAPIM/apim-charts/blob/stable/charts/portal/templates/jobs/cert-update-job.yaml#L18
best to have them consistent - either move labels under jobs too, or move nodeSelector/tolerations under portal